rollup.config.js

```diff
@@ -0,0 +1,31 @@
+import { terser } from "rollup-plugin-terser";
+
+function config({ input, format, minify, ext = "js" }) {
+  const dir = `build/`;
+  // const minifierSuffix = minify ? ".min" : "";
+  const minifierSuffix = "";
+  return {
+    input: `./src/js/${input}.js`,
+    output: {
+      name: "loadPyodide",
+      file: `${dir}/${input}${minifierSuffix}.${ext}`,
+      format,
+      sourcemap: true,
+    },
+    plugins: [
+      minify
+        ? terser({
+            compress: true,
+            mangle: true,
```

Comment:
Probably no need to mangle names, to save <1kB given the size of Pyodide?

Reply:
Okay.